src/core/blocks.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+
+use crate::core::hash::Sha3Hasher;
+use crate::core::header::Header;
+use crate::core::number::U256;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Block {
+    // @todo export Sha3 and U256 as generic
+    pub header: Header<Sha3Hasher, U256>,
```

Comment:
Again, U256 is overkill for block numbers.

Reply:
now, re-export block and header in mod.rs
```
pub type BlockNumU64 = u64;
pub type Sha3_256 = crate::core::hash::Sha3Hasher;
pub type Header = crate::core::header::Header<Sha3_256, BlockNumU64>;
pub type Block = crate::core::blocks::Block<Header>;
```